### ISSUE TITLE
Compatible with Pi Zero 2 W

### ISSUE THREAD

**Issue by ppent** [anerdins/nibepi#10]
It's difficult to get a Pi Zero W at the moment. Will this project also work with the newer version 1 W or is the power consumption too high with that model?
Thank you!

**Comment by Pete1979** [comment]
Its about impossible to get a Zero W, Compatibility with Zero 2 W is kinda needed.

**Comment by anerdins** [comment]
The newest image should be compatible with raspberry zero 2. No one has confirmed it though.Den 17 apr. 2022 22:48 skrev Pete1979 ***@***.***>:
Its about impossible to get a Zero W, Compatibility with Zero 2 W is kinda needed.

**Comment by anerdins** [comment]
No measurements has been made. Try research the power consumption of the new board. Den 9 apr. 2022 17:32 skrev ppent ***@***.***>:
It's difficult to get a Pi Zero W at the moment. Will this project also work with the newer version 1 W or is the power consumption too high with that model?
Thank you!

**Comment by steefan85** [comment]
I am using a Pi Zero 2 W with a F730 since January 2022 without any problems; works great!
No changes necessary wotks out of the box.

**Comment by ppent** [comment]
> I am using a Pi Zero 2 W with a F730 since January 2022 without any problems; works great! No changes necessary works out of the box.

Just to make sure: You don't use any external power supply, correct?

**Comment by steefan85** [comment]
> > I am using a Pi Zero 2 W with a F730 since January 2022 without any problems; works great! No changes necessary works out of the box.
> 
> 
> 
> Just to make sure: You don't use any external power supply, correct?

Yes, that's correct. I'm using the Wide Input Shim directly connected to the 12V Connection on the F730 Mainboard.

**Comment by MoYzOne1** [comment]
> > > I am using a Pi Zero 2 W with a F730 since January 2022 without any problems; works great! No changes necessary works out of the box.
> > 
> > 
> > Just to make sure: You don't use any external power supply, correct?
> 
> Yes, that's correct. I'm using the Wide Input Shim directly connected to the 12V Connection on the F730 Mainboard.

I have a Pi Zero 2 W too - revision 2.13 but the image doesn't work out of the box. The WiFi interface is missing. With a normal Buster image it works. Any ideas ? Maybe a missing driver or something. Is there a way to combine the NibePi image with a up to date buster ?

Greetz MoYz

**Comment by steefan85** [comment]
> > > > I am using a Pi Zero 2 W with a F730 since January 2022 without any problems; works great! No changes necessary works out of the box.
> > > 
> > > 
> > > Just to make sure: You don't use any external power supply, correct?
> > 
> > 
> > Yes, that's correct. I'm using the Wide Input Shim directly connected to the 12V Connection on the F730 Mainboard.
> 
> I have a Pi Zero 2 W too - revision 2.13 but the image doesn't work out of the box. The WiFi interface is missing. With a normal Buster image it works. Any ideas ? Maybe a missing driver or something. Is there a way to combine the NibePi image with a up to date buster ?
> 
> Greetz MoYz

Hi,

am running on Debian Stretch, so maybe my image-file [nibepi_1.1.rar](http://anerdins.se/NibePi/nibepi_1.1.rar) was not the same as yours.

The newest Version [nibepi_1.1.1.rar](https://github.com/anerdins/nibepi/blob/master/README.en.md) was built for Debian Bullseye

**Comment by MoYzOne1** [comment]
> > > > > I am using a Pi Zero 2 W with a F730 since January 2022 without any problems; works great! No changes necessary works out of the box.
> > > > 
> > > > 
> > > > Just to make sure: You don't use any external power supply, correct?
> > > 
> > > 
> > > Yes, that's correct. I'm using the Wide Input Shim directly connected to the 12V Connection on the F730 Mainboard.
> > 
> > 
> > I have a Pi Zero 2 W too - revision 2.13 but the image doesn't work out of the box. The WiFi interface is missing. With a normal Buster image it works. Any ideas ? Maybe a missing driver or something. Is there a way to combine the NibePi image with a up to date buster ?
> > Greetz MoYz
> 
> Hi,
> 
> am running on Debian Stretch, so maybe my image-file [nibepi_1.1.rar](http://anerdins.se/NibePi/nibepi_1.1.rar) was not the same as yours.
> 
> The newest Version [nibepi_1.1.1.rar](https://github.com/anerdins/nibepi/blob/master/README.en.md) was built for Debian Bullseye

I am indeed using the 1.1.1 image. But it should work too. Very strange. Thank you for your reply. I will try the Stretch image. If WiFi is running there, maybe I can upgrade to Bullseye from there.

**Comment by MoYzOne1** [comment]
> > > > > > I am using a Pi Zero 2 W with a F730 since January 2022 without any problems; works great! No changes necessary works out of the box.
> > > > > 
> > > > > 
> > > > > Just to make sure: You don't use any external power supply, correct?
> > > > 
> > > > 
> > > > Yes, that's correct. I'm using the Wide Input Shim directly connected to the 12V Connection on the F730 Mainboard.
> > > 
> > > 
> > > I have a Pi Zero 2 W too - revision 2.13 but the image doesn't work out of the box. The WiFi interface is missing. With a normal Buster image it works. Any ideas ? Maybe a missing driver or something. Is there a way to combine the NibePi image with a up to date buster ?
> > > Greetz MoYz
> > 
> > 
> > Hi,
> > am running on Debian Stretch, so maybe my image-file [nibepi_1.1.rar](http://anerdins.se/NibePi/nibepi_1.1.rar) was not the same as yours.
> > The newest Version [nibepi_1.1.1.rar](https://github.com/anerdins/nibepi/blob/master/README.en.md) was built for Debian Bullseye
> 
> I am indeed using the 1.1.1 image. But it should work too. Very strange. Thank you for your reply. I will try the Stretch image. If WiFi is running there, maybe I can upgrade to Bullseye from there.

Using the 1.1 image the Pi doesn't even boot (Rainbow Screen) ... original Rasbian Lite images boot with WiFi without any issues. I dont't understand this :(. I thought: hey: nice to have an image. Just write it to SD and your done. Seems like no NibePi for me.

**Comment by vytautasmiel** [comment]
To launch NibePi on Raspberry Pi Zero 2 W and get it connected via wifi I had to add firmware files brcmfmac43436-sdio.bin, brcmfmac43436-sdio.txt to /lib/firmware/brcm/ folder.

**Comment by MoYzOne1** [comment]
Thank you for your answer - Meanwhile I managed to make a clean install of Node Red importing the NibePi flows.
